src/looprestoration_tmpl_16.rs

```diff
@@ -2,6 +2,7 @@ use crate::include::stddef::*;
 use crate::include::stdint::*;
 #[cfg(all(feature = "asm", any(target_arch = "arm", target_arch = "aarch64"),))]
 use crate::src::align::Align16;
+use crate::src::mc::FnVersion;
```

Comment:
Maybe we should move `FnVersion` and `FnAsmVersion` to `src/cpu.rs`?

Reply:
Done!